components/plan-build/bin/hab-plan-build.sh

```diff
@@ -177,6 +177,18 @@
 # pkg_expose=(80 443)
 # ```
 #
+# ### pkg_exports
+# An associative array representing configuration data which should be gossiped to peers. The keys
+# in this array represent the name the value will be assigned and the values represent the toml path
+# to read the value.
+# ```
+# pkg_exports=(
+#   [key]=token
```

Comment:
What happens if we want to export something that's in the `sys` namespace, or the `pkg` namespace? The examples have unqualified names.

For example, the IP doesn't show up in `cfg`, it shows up in `sys`.